controllers/grid/settings/sections/SectionGridHandler.php

```diff
@@ -67,15 +69,16 @@ public function initialize($request, $args = null)
         $gridData = [];
         while ($section = $sectionIterator->next()) {
             // Get the section editors data for the row
-            $assignedSubEditors = $subEditorsDao->getBySubmissionGroupId($section->getId(), ASSOC_TYPE_SECTION, $journal->getId());
-            if (empty($assignedSubEditors)) {
+            $users = $subEditorsDao->getBySubmissionGroupIds([$section->getId()], ASSOC_TYPE_SECTION, $journal->getId());
```

Comment:
I think this is intentional, and probably a good thing, but just noting: this will now list users who were previously only auto-assigned, which might cause confusion for existing users who are used to the list *only* showing section editors. Strictly up to you, but if you think it's worth tweaking, it might be good to group the listed users by user group, e.g.:
> Editors: Barnes
> Section Editors: Buskins, Berardo

Reply:
It's a good idea, but I'm hesitant to try to fit so much info onto a grid row. Maybe something to look at when we redesign the section components.